.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+    "typescript.tsdk": "node_modules/typescript/lib"
+}
```

Comment:
Have you installed Editorconfig for your vscode? It supposed to catch the formatting issue from the https://github.com/WiseEngineering/crispy-cli/pull/6/commits/4cec172672010893a8907f79496b265c5896f5aa

Reply:
Editorconfig is catching this issue if you open the file but this file was generated by vscode.
Thanks for bringing it up